framework/src/test/java/org/apache/drill/test/framework/common/DrillTestConstants.java

```diff
@@ -0,0 +1,6 @@
+package org.apache.drill.test.framework.common;
```

Comment:
We have `framework/src/main/java/org/apache/drill/test/framework/DrillTestDefaults.java` 
Do you think we can integrate with that?

Reply:
Considered that before creating this class. The intention is to keep all TestNG related constants together and not mix it with `DrillTestDefaults`. Constant required by the framework as well, will go in `DrillTestDefaults`.

Reply:
Sounds good. May be rename it to indicate that? Say DrillTestNGDefaults? The same class name may be confusing.

Reply:
Done.